src/main/java/com/rtb/projectmanagementtool/task/TaskController.java

```diff
@@ -95,9 +96,22 @@ private boolean allSubtasksAreComplete(TaskData task) {
   }
 
   private void completeTask(TaskData task) {
-    task.setStatus(Status.COMPLETE);
-    if (task.getTaskID() != 0) {
-      datastore.put(task.toEntity());
+    if (task.getStatus() != Status.COMPLETE) {
+      task.setStatus(Status.COMPLETE);
+      if (task.getTaskID() != 0) {
+        datastore.put(task.toEntity());
+        UserController userController = new UserController(datastore);
+        UserData user;
+        for (long userID : task.getUsers()) {
+          try {
+            user = userController.getUserByID(userID);
+            user.setUserTotal(user.getUserTotal() + 1);
+            datastore.put(user.toEntity());
```

Comment:
I mentioned in one of Godfavour's PR, since `put` can accept a list of Entities, I wonder if we can pass it a single ArrayList containing both the Task and the Users that needs to be updated, so we only have to do 1 db call